src/main.ts

```diff
@@ -0,0 +1,63 @@
+import { setup } from '#lib/setup/all';
+import { registerCommands } from '#lib/utilities/register-commands';
+import { envParseInteger, envParseString } from '@skyra/env-utilities';
+import { Client, container } from '@skyra/http-framework';
+import { init, load } from '@skyra/http-framework-i18n';
+import '@skyra/shared-http-pieces/register';
+import { createBanner } from '@skyra/start-banner';
+import gradient from 'gradient-string';
+
+await setup();
+
+await load(new URL('../src/locales', import.meta.url));
+await init({ fallbackLng: 'en-US', returnNull: false, returnObjects: true, returnEmptyString: false });
+
+const client = new Client();
+await client.load();
+client.on('error', (error) => container.logger.error(error));
+
+await container.server.listen({
+	host: envParseString('API_ADDRESS'),
+	port: envParseInteger('API_PORT')
+});
+
+void registerCommands();
+
+const address = envParseString('HTTP_ADDRESS', '0.0.0.0');
+const port = envParseInteger('HTTP_PORT', 3000);
+await client.listen({ address, port });
+
+console.log(
+	gradient.morning.multiline(
+		createBanner({
+			logo: [
+				String.raw`            §`,
+				String.raw`▄▄▄▄▄▄▄▄▄▄▄▄║▄▄▄▄▄▄▄▄▄▄▄▄`,
+				String.raw`████████████║████████████`,
+				String.raw`║▄      ╚╬╬╬║╬╬╬╝      ▄║`,
+				String.raw`║█╗      ║║║║║║║      ╔█║`,
+				String.raw`╚╬█╗     ╚╣║║║╠╝     ╔█╬╝`,
+				String.raw` ║██╗▄▄▄▄▄║║║║║▄▄▄▄▄╔██║`,
+				String.raw`╔╝██║     ║║║║║     ║██╚╗`,
+				String.raw`║███║    ╔╝║║║╚╗    ║███║`,
+				String.raw`║███║    ║██║██║    ║███║`,
+				String.raw`║▀▀▀▀▀▀▀▀▀▀▀║▀▀▀▀▀▀▀▀▀▀▀║`
```

Comment:
No, it's the top part of the ArchId Ring — which Ring is.

Line 34 (omitted) is the ArchId Tower, the only thing visible from the outside.
Lines 35 and 36 are the lithosphere and the Layer L1.
Lines 37 to 43 represent the Ring Pillar, which is where computers are at, with the mantle supports on the sides, and a particle accelerator at line 40.
Line 44 represents the Layer L2.

| Nova's Spheres | ArchId Ring |
| --- | --- |
| ![Nova Spheres](https://user-images.githubusercontent.com/24852502/216570262-7a4681ea-e7d8-412c-b219-f012da7fa590.png) | ![Nova Spheres ArchId Ring](https://user-images.githubusercontent.com/24852502/216569230-27967428-5238-4fe3-a5b6-36499a603388.png) |

This is a fully hand-drawn logo, although it's compressed horizontally because the particle accelerator is *very* wide.